packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3387,7 +3387,7 @@ export type MessageThreadProps = {
     onDisplayDateTimeString?: (messageDate: Date) => string;
     mentionOptions?: MentionOptions;
     inlineImageOptions?: InlineImageOptions;
-    richTextEditor?: boolean;
+    richTextEditor?: boolean | /* @conditional-compile-remove(rich-text-editor-image-upload) */ RichTextEditorOptions;
```

Comment:
Has this been GA'd already?
If not, let's just replace the boolean.
If yes, can we change the rteOptions to a new parameter?

Reply:
As agreed with the team, I'll update `richTextEditor` to use `RichTextEditorOptions` as type